website/docs/d/search_deployment.html.markdown

```diff
@@ -1,37 +1,77 @@
 ---
 layout: "mongodbatlas"
-page_title: "MongoDB Atlas: search deployment"
-sidebar_current: "docs-mongodbatlas-datasource-search-deployment"
+page_title: "MongoDB Atlas: mongodbatlas_search_deployment"
```

Comment:
were these deleted/added headers wrong? should we change them in all resources / ds?

Reply:
I double-checked them with Zuhair and he confirmed that this metadata is no longer used by Hashicorp. Hashicorp used this metadata for screen readers which does not seem to be the case anymore.